src/config.c

```diff
@@ -691,7 +691,7 @@ void read_FTLconf(void)
 	// defaults to: DROP
 	buffer = parse_FTLconf(fp, "REPLY_WHEN_BUSY");
 
-	if(buffer != NULL && strcasecmp(buffer, "ACCEPT") == 0)
+	if(buffer != NULL && strcasecmp(buffer, "ALLOW") == 0)
```

Comment:
```suggestion
	if(buffer != NULL && (strcasecmp(buffer, "ALLOW") == 0 || strcasecmp(buffer, "ACCEPT") == 0))
```

We should accept both here to ensure we are not breaking existing configs where users found it they have to set `ACCEPT` instead of `ALLOW`.